test/app.js

```diff
@@ -20,7 +20,4 @@ beforeEach('drop tables, re-create them and populate sample data', done => {
 });
 
 // including all test files
-const normalizedPath = path.join(__dirname, '.');
-fs.readdirSync(normalizedPath).forEach(file => {
-  require(`./${file}`);
-});
+require('./controllers/user.js');
```

Comment:
The problem with this is that each new test file you add you should manually add it there. In case you wanna make it automatic you can use the following code:
```
const normalizedPath = path.join(__dirname, '.');
fs.readdirSync(normalizedPath).forEach((file) => {
  if (fs.lstatSync(`${normalizedPath}/${file}`).isDirectory()) {
    fs.readdirSync(`${normalizedPath}/${file}`).forEach((inFile) => {
      require(`./${file}/${inFile}`);
    });
  } else {
    require(`./${file}`);
  }
});
```